src/cb_admin/pages/Dashboard.js

```diff
@@ -51,23 +52,7 @@ export default class Dashboard extends React.Component {
   componentDidMount() {
     CommunityBusiness.get({ fields: ['name'] })
       .then(res => this.setState({ orgName: res.data.result.name }))
-      .catch((error) => {
-
-        if (ErrorUtils.errorStatusEquals(error, 401)) {
-          this.props.history.push('/cb/login');
-
-        } else if (ErrorUtils.errorStatusEquals(error, 500)) {
-          this.props.history.push('/error/500');
-
-        } else if (ErrorUtils.errorStatusEquals(error, 404)) {
-          this.props.history.push('/error/404');
-
-        } else {
-          this.props.history.push('/error/unknown');
-
-        }
-      });
-
+      .catch(err => redirectOnError(this.props.history.push, err, { 401: '/admin/login' }));
```

Comment:
`/admin/login` won't work, they need to log in again.

Reply:
Fixed in amend.